File: Material_de_apoyo/Sistemas_Distribuidos-master 29 nov/Practica 7 29_Agosto_2018/Prueba de rendimiento/Ejercicio_4.cpp
#include <iostream>
#include <stdlib.h>
#include <vector>
#include <string>
#include <cstring>
#include <math.h>

using namespace std;




int main(){


	double i = 0;
	int max = 100000000;

	double seno;
	double coseno;
	double tangente;
	double logaritmo;
	double raizCuadrada;


	while(i < max){

		coseno = coseno + cos(i);
		seno = seno + sin(i);
		tangente = tan(i);
		logaritmo = logaritmo + log(i);
		raizCuadrada = raizCuadrada + sqrt(i);

		i++;

	}


	cout << "Valor del seno : " << seno << endl;
	cout << "Valor del coseno : " << coseno << endl;
	cout << "Valor del tangente : " << tangente << endl;
	cout << "Valor del logaritmo : " << logaritmo << endl;
	cout << "Valor del raizCuadrada : " << raizCuadrada << endl;



	return 1;
}

File: Material_de_apoyo/Sistemas_Distribuidos-master 29 nov/Practica 7 29_Agosto_2018/Prueba de rendimiento/Ejercicio_4.java
import java.lang.Math;

public class Ejercicio_4 {

    public static void main(String[] args)
     {
       
     	double i = 0;
		int max = 100000000;

		double seno =0;
		double coseno = 0;
		double tangente =0;
		double logaritmo =0;
		double raizCuadrada =0;


		while(i < max){

			coseno = coseno + Math.cos(i);
			seno = seno + Math.sin(i);
			tangente = Math.tan(i);
			logaritmo = logaritmo + Math.log(i);
			raizCuadrada = raizCuadrada + Math.sqrt(i);

		i++;

	}

	System.out.println("Valor del seno : " + seno );
	System.out.println("Valor del coseno : " + coseno );
	System.out.println("Valor del tangente : " + tangente );
	System.out.println("Valor del logaritmo : " + logaritmo );
	System.out.println("Valor del raizCuadrada : " +raizCuadrada );




    }
}
